Source/GRChomboCore/ChomboParameters.hpp

```diff
@@ -73,24 +54,14 @@ class ChomboParameters
                     (boundary_params.lo_boundary[idir] ==
                      BoundaryConditions::REFLECTIVE_BC))
                 {
+                    // not both
```

Comment:
Will having reflective BCs in both directions break the interpolator? @JCAurre pointed out to me that making both directions reflective, whilst a bit unphysical, would actually allow you to simulate only half of a periodic box where that box had a reflective symmetry half way. So it is not impossible that a user might choose this. If we can't support it without a major change I think we should leave it, but if we can retain the option easily I think we should.

Reply:
Yeah from the top of my head I think that this should be fine. 

Of course there is the generic problem of what happens if the user asks for a point which has no reflection on the grid (error and crash), but I think that was already a problem from before and you probably want it to crash in order to know.

This is already deviating from your comment, but I'm thinking we should add an assertion checking the radii of extraction are all smaller than the box, as this is a common mistake I do.

Reply:
Ahhh Amelia's comment reminded me of why this was done. When we have reflective BC on the low boundary, we set center to 0 and in the high boundary we set it to the upper edge of the box. With both, there's not really a good choice. What do you think if this?

Reply:
I think in that case @amelialdrew wrote it so that it would leave the centre as L/2, which I think is ok and makes sense since there is not a preferred direction. It's definitely good to have the write out of the centre in the initial output, so that people can check it is what they expected.